internal/util/vault.go

```diff
@@ -184,21 +199,13 @@ func (kms *VaultKMS) GetPassphrase(key string) (string, error) {
 
 // SavePassphrase saves new passphrase in Vault.
 func (kms *VaultKMS) SavePassphrase(key, value string) error {
-	data, err := json.Marshal(map[string]map[string]string{
-		"data": {
+	data := map[string]interface{}{
+		"data": map[string]string{
 			"passphrase": value,
 		},
-	})
-	if err != nil {
-		return fmt.Errorf("passphrase request data is broken: %s", err)
 	}
 
-	resp, err := kms.request("POST", kms.getKeyDataURI(key), data)
-	if err != nil {
-		return fmt.Errorf("failed to POST passphrase for %s to vault: %s", key, err)
-	}
-	defer resp.Body.Close()
-	err = kms.processError(resp, "save passphrase")
+	err := kms.secrets.PutSecret(filepath.Join(kms.vaultPassphrasePath, key), data, kms.keyContext)
```

Comment:
just return an error? or do logging? currently, only error check is done

Reply:
oh, yes, just returning is sufficient, I think